### ISSUE TITLE
Map updates for mappers doesn't work with Devanagari and Eastern Arabic numerals

### ISSUE THREAD

**Issue by sanketgarade** [osmandapp/OsmAnd-iOS#2969]
I have 3 devices (2 running iOS and 1 with iPadOS) on which hourly updates are not getting activated despite me having -

1. More than 50 contributions in last 2 months. 
2. Having "applied" hourly updates to the maps I've already downloaded. 

It was working on one of the iOS devices for some days, but since last 4-5 days even that has stopped working. 

The eligibility page shows me having done 0 contributions. Please see below screenshots of the app and also of my hdyc stats. 
<img src="https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/20ed1783-c2ff-456e-bb81-f9ec82c28762"  width="400" height="auto"/> 

<img src="https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/aff6d651-b24a-4ae5-b3e1-877e19fe9cf0"  width="400" height="auto"/>

**Comment by sanketgarade** [comment]
Additionally here are snaps of the live update settings page.

![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/18030aad-a74e-4327-b18d-8fb67600525e)

![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/cd3e64b9-1c30-44a6-b0be-1b9ab52aa216)

**Comment by vshcherb** [comment]
Hello please share your OSM id for us to check

**Comment by sanketgarade** [comment]
> Hello please share your OSM id for us to check

Sure. I'd like to share in a private way. Is there any email or telegram username or any other way in which I can send it to you?

**Comment by vshcherb** [comment]
Please send it by to support email - https://osmand.net/ at the footer page

**Comment by vshcherb** [comment]
,"changesets":{"2023-07":20,"2023-06":30},"objectChanges":{"2023-07":555,"2023-06":939}} as I see but backend api you should see these numbers for last 2 months. Probably you need to logout and login.

Likely you don't see other changes cause you changed OSM name in June

**Comment by sanketgarade** [comment]
> Probably you need to logout and login

Have tried that multiple times and retried now again, but no change seen. Latest screenshots pasted below.

> Likely you don't see other changes cause you changed OSM name in June

ok. 



![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/9714464f-21fd-4dac-bad7-52cb0c2076f0)
![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/5ad8a041-f7cc-443e-b0ec-6367f6e1291a)
![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/bce0265a-469f-4e90-aef2-27992f651571)
![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/52e34e0d-0501-4d0e-810f-2b48ea1fddd9)

**Comment by sanketgarade** [comment]
Ok so I found the condition that makes this work or not work on my devices. The condition is the app language in the iOS settings. 

Language set to 

- Marathi - map contributions show up as zero and hence doesn't work.
- English - map contributions show up correctly and hence works.

**Comment by vshcherb** [comment]
Thanks for information! I think it's a good clue

**Comment by dmpr0** [comment]
<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/b0bb6157-7f76-419d-9d51-b7305780b2df"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/d80c3158-a24c-4946-b2f7-2c3d9f9ee06d"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
Probably bug is related to the numbers (2023-01) and not to the language. As you see it's not reproducible with latin written numbers.

**Comment by sanketgarade** [comment]
> Probably bug is related to the numbers (2023-01) and not to the language. As you see it's not reproducible with latin written numbers.

Yes. I set the app language to Marathi but this time set the numbers format to 123 instead of १२३. (See screenshot) And the contributions showed up normally. 

Maybe OsmAnd needs to force the number format (for this particular plugin/feature) to be 123 despite iOS setting set to something else. 

![image](https://github.com/osmandapp/OsmAnd-iOS/assets/74632379/cb942147-4006-4594-a301-5bc9bfa358b7)

**Comment by dmpr0** [comment]
<table style="width:100%">
  <tr>
    <th colspan=2>Indian numbering system</th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/e3bac7e3-5992-4aa3-a8a4-749c1848e328"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/e7ed8c5c-6818-4422-81d5-ce0bf22abfa5"  width="300" height="auto"/> </td>
  </tr>
</table>   

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/f8248daa-a31e-45f0-ac6b-3b9cfc84ff86"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/74a6dc7c-2aa3-4a23-99ff-3c02e545b7c1"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
The problem is in non-latin numbers, workaround: switch numbers to enable free version
